indico/web/templates/_session_bar.html

```diff
@@ -48,64 +48,76 @@
-            </div>
-            <select name="tz" class="tz-select" size="12" {{ 'disabled' if timezone_data.tz_mode != 'custom' }}>
-                {% for tz in timezone_data.timezones -%}
-                    <option value="{{ tz }}" {{ 'selected' if tz == session.tzinfo.zone }}>{{ tz }}</option>
-                {%- endfor %}
-            </select>
-            {% if session.user %}
-                <div class="tz-update-user-container">
-                    <input type="checkbox" name="update_user" id="tz-update-user" value="1">
-                    <label for="tz-update-user">
-                        {%- trans %}Remember these settings{% endtrans -%}
+
+                <form action="{{ url_for('core.change_tz') }}" method="post">
+                    <fieldset id="tz-modes">
+                        <legend>Use timezone based on:</legend>
+                        <input type="radio" name="tz_mode" id="tz-mode-local" value="local"
+                               {{ 'checked' if timezone_data.tz_mode == 'local' }}
+                               aria-label="{% trans %}Use the event/category timezone{% endtrans %}">
+                        <label for="tz-mode-local">{% trans %}Event/category{% endtrans %}</label>
+                        {% if session.user %}
+                            <input type="radio" name="tz_mode" id="tz-mode-user" value="user"
+                                   data-user-tz="{{ profile_timezone }}"
+                                   {{ 'checked' if timezone_data.tz_mode == 'user' }}
+                                   aria-label="{% trans %}Use the default timezone of your profile{% endtrans %}">
+                            <label for="tz-mode-user">
+                                {% trans %}Profile{% endtrans %}
+                            </label>
+                        {% endif %}
+                        <input type="radio" name="tz_mode" id="tz-mode-custom" value="custom"
+                               {{ 'checked' if timezone_data.tz_mode == 'custom' }}
+                               aria-label="{% trans %}Use a custom timezone{% endtrans %}">
+                        <label for="tz-mode-custom">{% trans %}Custom{% endtrans %}</label>
+                    </fieldset>
+                    <label id="tz-custom-field">
+                        <span>{% trans %}Select a custom timezone{% endtrans %}</span>
+                        <select name="tz" size="12">
+                            {% for tz in timezone_data.timezones -%}
+                                {% set is_profile_timezone = tz == profile_timezone %}
+                                <option value="{{ tz }}" {{ 'selected' if tz == session.tzinfo.zone }} {{ 'data-profile' if is_profile_timezone }}>
+                                    {{ tz }}
+                                    {% if is_profile_timezone -%}
+                                        {# Translators: example usage: "Europe/Geneva (your profile timezone)" #}
```

Comment:
Nobody will read that ;x

I suggest doing this instead (untested):

```jinja
                                    {% if is_profile_timezone -%}
                                        {% trans %}{{ tz }} (your profile timezone){% endtrans %}
                                    {%- else -%}
                                        {{- tz -}}
                                    {%- endif %}                                    
```

That way the placeholder is in the translatable string so it'll be much clearer how it's used

Reply:
Ah, great idea! Thanks